functions-methods.js

```diff
@@ -33,4 +60,16 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+
+    if (email.includes('@') === true && email.includes(',') != true && !/[.]$/.test(email)) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Je kunt ook direct de uitkomst teruggeven
```suggestion
    return email.includes('@') === true && email.includes(',') != true && !/[.]$/.test(email);
```